src/Classes/iTones/iTones_Utils.php

```diff
@@ -357,4 +375,13 @@ public static function telnetEnd()
             self::$telnet_handle = null;
         }
     }
+
+    private static function log(...$fields)
```

Comment:
Ideally this would have a config var for turning it on / off and a logging location.